split-sub-dir-into-new-repo.md

```diff
@@ -10,11 +10,14 @@ Based from [github help link](https://help.github.com/en/articles/splitting-a-su
     $ git commit --allow-empty -m "root commit"
 ```
 3. Determine the branch and sub-dir to split, and split it.
+   NOTE: sub/dir/to/split refers to the repo location, not local file.  Use / not \
```

Comment:
i addressed this in
```
5bef551 ("split-repo: Clarify step for any branch", 2020-08-22)
```
by using `sub/dir/1/to/split/` for dir (added trailing slash to denote a dir).